src/config/themeMode/themeModeStorage.tsx

```diff
@@ -0,0 +1,10 @@
+import { THEME_MODE_KEY } from '../../constants/storage';
+
+export const getIsDarkMode = () => {
```

Comment:
if that's getter i would name it getTheme and then check neccessary if we'd have more than dark|light i.e getTheme === 'dark' ?
If you want to have theme specific check function I would call it isDarkTheme()